### PR TITLE
Adding composer install examples

### PR DESCRIPTION
**Pull request by leggetter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,18 @@ Register at <https://pusher.com> and use the application credentials within your
 
 You can get the Pusher PHP library via a composer package called `pusher-php-server`. See <https://packagist.org/packages/pusher/pusher-php-server>
 
+```
+$ composer require pusher/pusher-php-server
+```
+
+Or add to `composer.json` and `composer update`:
```
Comment (zimbatm) [inline README.md]: and *then run* `composer update` ?

Comment (tanuck) [inline README.md]: also add a version constraint to the cli based `composer require`

Comment (vinkla) [inline README.md]: We could let composer handle the version it self for the CLI based example (like we already do).

**Comment by leggetter** [comment]
Thanks for the feedback. Made changes.

**Comment by zimbatm** [comment]
LGTM, not sure why Travis-CI failed

**Pull request merged**